### ISSUE TITLE
Undefined symbol: _OBJC_CLASS_$_MSPLCrashReporter

### ISSUE THREAD

**Issue by shyamjimishra1993** [microsoft/appcenter-sdk-apple#1621]
Hi ,

I am adding AppCenter by adding their .xcodeproj directly to our project and not the frameworks.
So i added AppCenter.xcodeproj and AppCenterCrashes.xcodeproj .
Then did the required linking steps like adding to UserHedare search path in build setting and adding target dependencies.

I am getting a liking issue saying-

Undefined symbol: _OBJC_CLASS_$_MSPLCrashReporter
Undefined symbol: _OBJC_CLASS_$_MSPLCrashReport
Undefined symbol: _OBJC_CLASS_$_MSPLCrashReporterConfig

Looks like I am missing something. I took the codebase from https://github.com/microsoft/appcenter-sdk-apple.

Any help on this would be really appreciated.

<img width="601" alt="Screenshot 2019-05-15 at 3 24 10 PM" src="https://user-images.githubusercontent.com/20599875/57767158-5c267e80-7726-11e9-941b-82aaca4a56b9.png">>

Thanks and Regards
Shyam Ji Mishra

**Comment by annakocheshkova** [comment]
@shyamjimishra1993 any reason you don't want to use the framework directly?
Looks like a reference to CrashReporter framework (PLCrashReporter) got broken. Try adding it manually from `Vendor/iOS/PLCrashReporter`

**Comment by shyamjimishra1993** [comment]
Hi @annakocheshkova,
No specific reason as such. Just wanted to understand the internals fo AppCenter better. So thought to integrate this way. Ok, Let me try importing CrashReporter.

**Comment by annakocheshkova** [comment]
Hey @shyamjimishra1993 did you manage to make it work?

**Comment by izackp** [comment]
If anyone else encounters this error, it is because the CrashReporter project inside of the AppCenterCrashes project is being built for the wrong architecture.

In my case, I have an m1 Mac with the simulator as my target (arm64), and the library was being built for x86_64.

The fix for me:

1. Open build settings for CrashReporter inside of the AppCenterCrashes
2. Update PL_SIMULATOR_ARCHS from `i386 x86_64` to `i386 x86_64 arm64`

Looks like there is already a branch to fix this in the PLCrashReporter repo.

**Comment by DmitriyKirakosyan** [comment]
Hi @izackp

Thank you for reaching out to us!
As you correctly noted, the fix is already prepared in this PR: https://github.com/microsoft/plcrashreporter/pull/185

**Comment by apps4everyone** [comment]
any news here? maybe a patch release? thx

**Comment by DmitriyKirakosyan** [comment]
Hi @apps4everyone , we are planning to release Apple SDK this week.

**Comment by apps4everyone** [comment]
@DmitriyKirakosyan thx, Great News

**Comment by Tj3n** [comment]
@DmitriyKirakosyan What about `MSACErrorReport`? I have to install AppCenter manually on my project, but I got this `AppCenterCrashes.framework/AppCenterCrashes(MSACErrorReport.o), building for iOS Simulator, but linking in object file built for iOS, file 'AppCenterCrashes.framework/AppCenterCrashes' for architecture arm64`.

 I've updated to [4.3.0](https://github.com/microsoft/appcenter-sdk-apple/releases/tag/4.3.0) but issue remain. Seems like need to rebuild that also?
